packages/mobile/src/components/Chat/ChatDetail.tsx

```diff
@@ -311,20 +310,19 @@ export const ChatDetail = memo(
           mode: 'open',
         });
         document.fileCopyUri = fixDocumentURI(document.fileCopyUri || document.uri);
-        const asset: Asset = {
+        const asset: ImageSource = {
```

Comment:
You should be able to use your helper method here again as well right? `assetsToImageSource`

Reply:
Not really as we convert type assets to image source. This one is a documentPickerResponse and really doesn't have anything that suits it